### ISSUE TITLE
vtm strategy.

### ISSUE THREAD

**Issue by saarsalu** [binary-com/binary-bot#2298]
Hi could anyone write such a bot

the video explains the whole strategy

https://files.fm/u/96v9adgt

Introducing the Volatility Traders Minds Strategy (VTM Strategy)
This is the most detailed volatility 75 index / other indices strategy available now so you can build that income you desperately want through trading.

This strategy will make you enter the market with complete confidence, with a surprisingly clear advantage over the masses and the amazing thing about this strategy, it is compatible with your smart phone and laptop as well, so you can trade anywhere.

**Comment by sohailsb** [comment]
Suspicious contents, be careful

**Comment by saarsalu** [comment]
> Suspicious contents, be careful

what you have to be careful about? the video has been uploaded via this link because it is too big to add as a zip file

**Comment by saidforex26** [comment]
@saarsalu I can make a strategy into a bot. please upload to Google Drive

**Comment by sohailsb** [comment]
> @saarsalu I can make a strategy into a bot. please upload to Google Drive

can you create bot on my strategy, its very profitable.
https://github.com/binary-com/binary-bot/issues/2296

**Comment by saarsalu** [comment]
> > @saarsalu I can make a strategy into a bot. please upload to Google Drive
> 
> can you create bot on my strategy, its very profitable.
> #2296

why are you under my post with your post?

**Comment by saarsalu** [comment]
> @saarsalu I can make a strategy into a bot. please upload to Google Drive

here you go mate!
https://drive.google.com/file/d/1CR-w26KY0pAMV6kOIAf8hIEGsru4Fc11/view?usp=sharing

**Comment by muroyal** [comment]
@saidforex26 I think ADX and MACD is in this strategy.. Are you able to add ADX indicator in the bot. 

Can you please provide any bot sample which have ADX ?

**Comment by alanvito1** [comment]
Oi, sem querer me intrometer aqui, já me intrometendo, tenho alguns bots com as funções que vocês falaram, são todos gratuitos, só baixar e estudar, analisar e fazer uma boa engenharia reversa.

Visite minha página para download de novos bots gratuitos.
Visit my librarie for new free bots.

https://github.com/alanvito1/superfree-binary-bot

**Comment by saidforex26** [comment]
> 
> 
> > @saarsalu I can make a strategy into a bot. please upload to Google Drive
> 
> can you create bot on my strategy, its very profitable.
> #2296

please develop
[differ00digit.zip](https://github.com/binary-com/binary-bot/files/4023720/differ00digit.zip)

**Comment by saidforex26** [comment]
> 
> 
> @saidforex26 I think ADX and MACD is in this strategy.. Are you able to add ADX indicator in the bot.
> 
> Can you please provide any bot sample which have ADX ?

@muroyal this is an adx indicator sample. please develop your strategy
[@indikator adx.zip](https://github.com/binary-com/binary-bot/files/4023739/%40indikator.adx.zip)

**Comment by muroyal** [comment]
@saidforex26 I am unable to load indicator. Getting unable to load block file. 

error can you pelase do need full..

**Comment by saarsalu** [comment]
@saidforex26 
How its going

I gave the video another shot and there’s more info I gathered. Here are the guidelines:

For Buys:
1. Price must be above 48 Ema
2. Candle must close above middle bollinger band
3. Macd must be above 0 at the time candle closed above middle band OR can wait until 2nd candle to close above 0 if it’s below 0 at the time candle closed above middle band. (Otherwise don’t take trade)
4. ADX must be above 0
-Close trade when candle closes above upper band (TP)
-Close trade when candle closes below middle band (SL)

For Sells:
1. Price must be below 48 Ema
2. Candle must close below middle bollinger band
3. Macd must be below 0 at the time candle closed Below middle band OR can wait until 2nd candle to close below 0 if it’s above 0 at the time candle closed below middle band. (Otherwise don’t take trade)
4. ADX must be above 0
-Close trade when candle closes below lower band (TP)
-Close trade when candle closes above middle band (SL)
